src/docs/arch-data-file.dox

```diff
@@ -1,6 +1,6 @@
 /*! @arch_page arch-data-file Data File Format
 
-@section Database Files
+@section database_files Database Files
```

Comment:
To make it consistent, can you start all tags with the same prefix: `data_file_xxx`? This will make it easy to cross-reference the sections to other pages and ensure there is no namespace conflict if someone else uses the same names for sections.

This is applicable to all section names below that do not start with `data_file_xxx` prefix.

Reply:
Sure, I will fix this.